### PR TITLE
add enviroment variables to README

### LINKED ISSUE #191: README environment variables
### Describe the feature

- add default environment variables to README
- explain the environment variables

### Why is this feature needed?

documention

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #191

### CODE REVIEW

**Review by florian-sabonchi — APPROVED** [review]

**Comment by DasProffi** [comment]
@MaxSchaefer do you have any additions?

**Comment by florian-sabonchi** [comment]
> @MaxSchaefer do you have any additions?

No was a mistake

**Comment by MaxSchaefer** [comment]
The advice to look up and copy the `.env.example` to a proper `.env` is good as a part of the README. But we should add a proper `.env.example` to every service instead of the listing in the README.

**Comment by DasProffi** [comment]
@MaxSchaefer is this okay or did I forget something?

**Review by FoseFx — APPROVED** [review]

File: services/user-svc/.env.example [hunk 1]
```diff
@@ -10,9 +9,19 @@ LOG_LEVEL=debug
 # The port on which the the grpc service will listen on (default: 8080)
 PORT=8080
 
+# Database Related Configurations
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=postgres
```
Comment (MaxSchaefer) [inline services/user-svc/.env.example:17]:
```suggestion
POSTGRES_DB=user-svc
```

File: services/task-svc/.env.example [hunk 2]
```diff
@@ -17,9 +16,12 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=postgres
```
Comment (MaxSchaefer) [inline services/task-svc/.env.example:17]:
```suggestion
POSTGRES_DB=task-svc
```

File: services/ory-svc/.env.example [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
+# Database Related Configurations
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=postgres
+
+# For development purposes
+INSECURE_FAKE_TOKEN_ENABLE=true
+OAUTH_CLIENT_ID=425f8b8d-c786-4ff7-b2bf-e52f505fb588
```
Comment (MaxSchaefer) [inline services/ory-svc/.env.example:21]:
```suggestion
```

This service doesn't need any database or auth

File: services/emergency-room-svc/.env.example [hunk 4]
```diff
@@ -13,7 +12,11 @@ PORT=8080
 # Database Related Configurations
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
-POSTGRES_USER=helpwave
-POSTGRES_PASSWORD=helpwave
-POSTGRES_DB=emergency-room-svc
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=postgres
```
Comment (MaxSchaefer) [inline services/emergency-room-svc/.env.example:17]:
```suggestion
POSTGRES_DB=emergency-room-svc
```

File: README.md [hunk 5]
```diff
@@ -104,6 +104,34 @@ you arguments into migrate/migrate running in a container.
 - Migrate the task-svc database one down: `./migrate.sh task-svc down 1`
 - Current migration version of the task-svc database: `./migrate.sh task-svc version`
 
+## Enviroment variables
+Every service should contain a `.env.example` file which should be copied to a usable `.env` file
+
+Default Variables:
```
Comment (MaxSchaefer) [inline README.md:110]: Should we keep the general overview of common environment variables?

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

**Comment by MaxSchaefer** [comment]
Do too https://github.com/helpwave/services/blob/main/libs/common/auth.go#L77, it should be possible to get rid of `OAUTH_CLIENT_ID` entirely or set a default value of "" instead of our real client id.

File: README.md (hunk 5 above)
Comment (DasProffi) [reply README.md:110]: I would say so having more information available should be better than fewer, but if we notice it's too confusing we can always remove it. So I would keep it for now

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**